cibuildwheel/__main__.py

```diff
@@ -136,11 +136,19 @@ def main():
     )
 
     if platform == 'linux':
-        manylinux1_x86_64_image = os.environ.get('CIBW_MANYLINUX1_X86_64_IMAGE', None)
-        manylinux1_i686_image = os.environ.get('CIBW_MANYLINUX1_I686_IMAGE', None)
+        # TODO Check CIBW_MANYLINUX1_X86_64_IMAGE and CIBW_MANYLINUX1_I686_IMAGE environment variables and error if they exist
+        # TODO Check CIBW_BUILD and CIBW_SKIP for "manylinux1" and error if present
+        # TODO Add documentation on CIBW_ENVIRONMENT and possibility of AUDITWHEEL_PLAT if not defined by custom manylinux image
+        manylinux_x86_64_image = os.environ.get('CIBW_MANYLINUX_X86_64_IMAGE', '2010')
+        manylinux_i686_image = os.environ.get('CIBW_MANYLINUX_I686_IMAGE', '')
+
+        default_manylinux_images_x86_64 = {'1': 'quay.io/pypa/manylinux1_x86_64',
+                                           '2010': 'quay.io/pypa/manylinux2010_x86_64'}
+        default_manylinux_images_i686 = {'1': 'quay.io/pypa/manylinux1_i686'}
```

Comment:
`CIBW_MANYLINUX_X86_64_IMAGE=1` looks like you're enabling a boolean option, or something numeric, but it's an identifier, so let's use `CIBW_MANYLINUX_X86_64_IMAGE=manylinux1` (and `manylinux2010`).

Reply:
Yes, ok, good point. Something started feeling lazy about using `'1'` and `'2010'`, so I agree!